### PR TITLE
Add activePath

### PR DESCRIPTION
**Pull request by jdlehman**
The activePath is the matching regex string that was set on the Switch's path prop (or null if one does not match)

- Passed as last arg of onChange callback
- Passed as last arg of renderSwitch callback
- Set as prop of component being rendered (component is now aware of why it was rendered based on the activePath)

### CODE REVIEW

**Comment by jdlehman** [comment]
cc @whatknight

File: src/helpers.js [hunk 1]
```diff
@@ -44,6 +44,18 @@ export function getSwitch(path, {children, basePath}) {
   })[0] || null;
 }
 
+export function getActivePath(currentPath, basePath, currentSwitch) {
+  if (!currentSwitch) { return null; }
```
Comment (whatknight) [inline src/helpers.js]: @jdlehman what do you think about returning `undefined` instead of `null`?

Comment (jdlehman) [inline src/helpers.js]: I am fine either way, `null` feels more explicit, but `undefined` is more like it wasn't set ever.  Hmm...

Comment (whatknight) [inline src/helpers.js]: its prob fine as null  ¯\_(ツ)_/¯

**Pull request merged**